spec/furniture/marketplace/menu/product_component_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe Marketplace::Menu::ProductComponent, type: :component do
+  subject(:output) { render_inline(component) }
+
+  let(:operator) { create(:person, operator: true) }
+
+  let(:component) { described_class.new(product:, cart:, current_person: operator) }
+  let(:marketplace) { create(:marketplace) }
+  let(:cart) { marketplace.cart_for_shopper(shopper: create(:marketplace_shopper)) }
+  let(:product_description) do
+    <<~DESC.chomp
+      A delicious fritter made with love
+
+      Make sure to **eat it warm!**
+    DESC
+  end
+  let(:product) { create(:marketplace_product, description: product_description, marketplace:) }
+
+  it { is_expected.to have_content(vc_test_controller.view_context.humanized_money_with_symbol(product.price)) }
+
+  it "renders the description as markdown" do
+    expect(output).to have_css("p", text: "A delicious fritter made with love")
+    expect(output).to have_css("strong", text: "eat it warm!")
```

Comment:
Is there an advantage to expecting on each part rather than doing something like: 
```ruby
expect(output).to eq(<p>A delicious fritter made with love</p> Make sure to <strong>eat it warm!<strong>
```

Reply:
Eh, mostly for "flexibility"; since if we switch from say, strong to italic; the A delicious fritter made with love bit shouldn't be sad.